test/test_queue_monkeypatch.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+
+import unittest
+
+import urllib3
+from urllib3.exceptions import EmptyPoolError
+import Queue
```

Comment:
This is probably going to break on Python 3 because the Queue module moved. You'll need to import it from six.